homepage/tests.py

```diff
@@ -133,17 +133,45 @@ def test_register(self):
 
     # # this will test for uniqness of username if tried to login with same username again page dont redirect
     def test_for_unique_username(self):
-        responce = self.client.post(reverse('homepage:register'),self.duplicate, follow=True)
-
-        self.assertContains(responce,'A user with that username already exists.',status_code=200)
+        responce = self.client.post(reverse('homepage:register'), self.duplicate, follow=True)
```

Comment:
check for typo errors to. It is `response` and not `responce`. Not mandatory to correct it. But just note it